packages/sanity/src/core/studio/screens/ConfigErrorsScreen.tsx

```diff
@@ -1,44 +0,0 @@
-import React from 'react'
-
-// const Root = styled(Card).attrs({height: 'fill', overflow: 'auto'})``
-// const Content = styled(Box).attrs({padding: 4})`
-//   margin: auto;
-//   width: 640px;
-//   max-width: 100%;
-// `
-
-export function ConfigErrorsScreen() {
-  return <>TODO: implement config errors screen</>
-  // return (
-  //   <Root forwardedAs={Flex}>
-  //     <Content forwardedAs={Flex} direction="column" gap={4}>
-  //       <Flex direction="column" gap={2}>
-  //         <Box>
-  //           <Heading as="h1">Configuration Error</Heading>
-  //         </Box>
-  //         <Box muted>
-  //           <Text>An error occurred while trying to resolve your Studio's configuration.</Text>
-  //         </Box>
-  //         <Box>
-  //           <Text size={1} muted>
-  //             Note: the design of this page may change.
-  //           </Text>
-  //         </Box>
-  //       </Flex>
-  //       <Card shadow={1}>
-  //         {errors.map((errorInfo, index) => (
-  //           // eslint-disable-next-line react/no-array-index-key
-  //           <ErrorMessage key={index} {...errorInfo} />
-  //         ))}
-  //       </Card>
-  //       <Button
-  //         tone="primary"
-  //         // eslint-disable-next-line react/jsx-no-bind
-  //         onClick={() => window.location.reload()}
-  //         type="button"
-  //         text="Retry"
-  //       />
-  //     </Content>
-  //   </Root>
-  // )
```

Comment:
I just noticed that this is actually used on StudioProvider. While I wasn't able to trigger this message to appear in the way that the issue described it, I feel that then the best approach here would be to put the interface that is commented out instead? 

Because the TODO just looks bad :/ Unless we have a different approach that was left along the way? @hermanwikner

Reply:
This screen can't be displayed at this time because it's commented out here, and if there's a configuration error we currently fall back to the default error screen: https://github.com/sanity-io/sanity/blob/5abfeccd0e7d861c4a52eb7e6103d9b3cd710b11/packages/sanity/src/core/studio/workspaceLoader/WorkspaceLoader.tsx#L162

The commented JSX might work but I think it needs a bit more work and design to match other error screens we have! I like that we clean up unused code but we should implement this screen at some point, so I think it might be nice to still keep this code until we start that work :)

Reply:
I'm wondering if it can't though :/ 
Because when I looked up [ConfigErrorsScreen.tsx](https://github.com/sanity-io/sanity/pull/4101/files/41d435e79070b926f9516227ad9648efda497905#diff-c27b006e5e8002d9e4bc0214c11e2ff00166b6ece3607618d7b23504d78d86da) I found it being used in the `StudioProvider`.

https://github.com/sanity-io/sanity/blob/053e47b66d118cb7952b92e500d7e051d96a3105/packages/sanity/src/core/studio/StudioProvider.tsx#L75

I haven't been able to replicate this appearing naturally while using the product but this screen appearing right now in any instance would look really bad. Do you know if it's a mistake that this screen is being used and something else should be used instead?

Reply:
The `ConfigErrorsScreen` component is passed to the `WorkspaceLoader`'s prop called `ConfigErrorsComponent`. But `WorkspaceLoader` do not render that component (since it is commented out) – so it can't be rendered currently!

Reply:
Sigh of course 🤦 
I don't even have an excuse for missing this 😂 

Great, then I think I'll add a comment on [ConfigErrorsScreen.tsx](https://github.com/sanity-io/sanity/pull/4101/files/41d435e79070b926f9516227ad9648efda497905#diff-c27b006e5e8002d9e4bc0214c11e2ff00166b6ece3607618d7b23504d78d86da) just about the plan to eventually clean it up and create a shortcut task :)

Reply:
@hermanwikner created a story on shortcut and added a comment for more visibility. if you're good with it we can go with a ✅